unified-scheduler-pool/src/lib.rs

```diff
@@ -1087,17 +1086,15 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
                         is_finished = session_ending && state_machine.has_no_active_task();
                     }
 
-                    if session_ending {
-                        state_machine.reinitialize();
-                        session_result_sender
-                            .send(std::mem::replace(
-                                &mut result_with_timings,
-                                initialized_result_with_timings(),
-                            ))
```

Comment:
this `std::mem::replace(...)` was calling extra `::initialized_result_with_timings()` per session wastefully..